benchmarks/define_arguments.py

```diff
@@ -0,0 +1,86 @@
+from skimage import restoration, data
+import numpy as np
+
+l1 = 1000
+
+img = np.random.randint(256, size=(l1, l1), dtype=np.uint8)
+img_rgb = np.random.randint(256, size=(l1, l1, 3), dtype=np.uint8)
+img_binary = data.binary_blobs(length=l1, volume_fraction=0.3).astype(np.uint8)
+img2 = data.binary_blobs(length=l1, volume_fraction=0.3, seed=10).astype(np.uint8)
+
+
+parameters = {
```

Comment:
I like this. It's a pretty nice, compact way to do this.

Reply:
Cool. I'm indeed looking for a middle ground where we can test as many functions as possible but without writing too much code for this.